doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -498,6 +498,50 @@ in :ref:`canonicalstructures`; here only a simple example is given.
    Otherwise said, :token:`qualid` is canonically used to extend the field |c_i|
    into a complete structure built on |c_i|.
 
+   The following kinds of terms are supported for the fields |c_i| of :token:`qualid`:
+
+   * Constants (defined through :cmd:`Definition`, :cmd:`Inductive` and constructors thereof,
+     :cmd:`Record` and fields thereof, etc.) and section variables of an active section,
+     applied to zero or more arguments.
+   * Any :token:`@sort`.
```

Comment:
You can use the ``` :n:`@sort` ``` syntax or the ``` :token:`sort` ``` syntax. The two are equivalent (the `:n:` syntax being more powerful because you can create expressions with multiple tokens). But here you are mixing the two and it doesn't make sense (and won't compile).